.circleci/config.yml

```diff
@@ -684,15 +684,6 @@ defaults:
       binary_type: native
       image: cimg/node:18.11
 
-  - job_ems_test_ext_colony: &job_ems_test_ext_colony
```

Comment:
Oh, one more thing. It would be best to have a short comment on the compile job that remains in code that says why we decided not to run tests for Colony. So that anyone looking at it later does not have to guess.

Reply:
Oh, I guess I was to late to do the change. It got merged already. Anyway, I can add the comment in another PR. I just noticed that from time to time (as for example in this PR), the CI is failing with the following: https://app.circleci.com/pipelines/github/ethereum/solidity/29819/workflows/8bf56b71-d3c4-48ca-ab74-4264020b4886/jobs/1324702?invite=true#step-103-17

```
curl: (22) The requested URL returned error: 403
```

I suspect that this is due to the fact that we are not doing authenticated request to github in this specific CI job, and consequently hitting the limit of request per hour that Github imposes on unauthenticated requests (see: https://docs.github.com/en/rest/overview/authenticating-to-the-rest-api?apiVersion=2022-11-28). If this is the case, it is easy to fix (just add the `Authorization` header).